sqrl-tools/sqrl-test/src/main/java/com/datasqrl/DatasqrlTest.java

```diff
@@ -174,29 +175,25 @@ public int run() {
       for (Exception e : exceptions) {
         if (e instanceof SnapshotMismatchException) {
           SnapshotMismatchException ex = (SnapshotMismatchException) e;
-          logRed("Snapshot mismatch for test: " + ex.getTestName());
-          logRed("Expected: " + ex.getExpected());
-          logRed("Actual  : " + ex.getActual());
+          log.error("Snapshot mismatch for test: {}\nExpected: {}\nActual  : {}",
+              ex.getTestName(), ex.getExpected(), ex.getActual());
           exitCode = 1;
         } else if (e instanceof JobFailureException) {
-          JobFailureException ex = (JobFailureException) e;
-          logRed("Flink job failed to start.");
-          ex.printStackTrace();
+          log.error("Flink job failed to start.", e);
           exitCode = 1;
         } else if (e instanceof MissingSnapshotException) {
           MissingSnapshotException ex = (MissingSnapshotException) e;
-          logRed("Snapshot on filesystem but not in result: " + ex.getTestName());
```

Comment:
This appears to break our color formatting. This is really helpful to see. We spoke about maybe logging twice so it can be easier to find in the build logs, where we could log when we observe a test failure and the produce a color summary at the end.